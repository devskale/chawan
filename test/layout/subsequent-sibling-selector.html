<!DOCTYPE html>
<style>
div ~ div { color: red }
</style>
<div>white</div>
<span>test</span>
<div>red</div>
