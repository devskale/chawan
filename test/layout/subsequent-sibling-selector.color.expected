white
test
[38;2;255;40;40mred[39m
